src/action/method.ts

```diff
@@ -115,9 +127,31 @@ export class AbstractMethod {
       selp = await this.signAction(envelop);
     }
 
-    await this.client.sendAction({
-      action: selp.action()
-    });
+    try {
+      await this.client.sendAction({
+        action: selp.action()
+      });
+    } catch (e) {
+      let code = ActionErrorCode.ErrUnknown;
+      let message = `send action error: ${JSON.stringify(e)}`;
+      if (e.details) {
+        message = e.details;
```

Comment:
can we parse the exact object like e.details[0].Description === "err string"?

using starts with and ends with looks strange